src/controllers/root_controller.js

```diff
@@ -19,10 +19,12 @@ const RootController = express.Router();
 /**
  * Get the home page.
  */
-RootController.get('/', TunnelSetup.isTunnelSet, function(request, response) {
-  response.sendFile('index.html', {
-    root: Constants.BUILD_STATIC_PATH,
-  });
-});
+
```

Comment:
Why the changes here?

Reply:
The Lint checker on my machine wants to see 'x' number of spaces between one line and the next. To be honest, the line space thing on javascript ..... is 'different' to what I was expecting. :-/ ... they should copy the python dudes on this!